src/app/pages/FiatOnrampPage/index.tsx

```diff
@@ -112,10 +112,9 @@ export function FiatOnramp() {
           <iframe
             height="875"
             title="Transak On/Off Ramp Widget"
-            // From https://github.com/Transak/transak-sdk/blob/2ebb3bd/src/index.js#L52
-            // TODO: It also loads https://api.sardine.ai inside, and that tries
-            // to listen for devicemotion and deviceorientation. Is this necessary?
-            allow="camera;microphone;fullscreen;payment"
+            // Expands on https://github.com/Transak/transak-sdk/blob/2ebb3bd/src/index.js#L52
+            // and somewhat matches https://docs.transak.com/docs/web-integration#embediframe-webapp
+            allow="accelerometer;camera;microphone;fullscreen;gyroscope;payment"
```

Comment:
this is weird why they need motion detection

Reply:
It could be for video liveness check (does device movement match the video), but the event listeners get attached much earlier by https://api.sardine.ai/assets/loader.min.js

I'm leaning towards increasing permissions, since we don't have insight into whether this decreases users' ability to complete KYC